grammars/javascript.cson

```diff
@@ -1218,6 +1218,31 @@
         ]
       }
       {
+        'begin': '(Relay\\.QL)\\s*(`)'
```

Comment:
Upon further discussion we've decided to not do this for now, as the entire `Relay.QL` is a marker saying that the contents of the template string is GraphQL source. We can always capture the `.` separately later on if we decide we want to.